my-app/src/components/HomesGuestLoves/Container.js

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import { SwiperSlide } from 'swiper/react';
+import HomesGuestLovesHeader from './HomesGuestLovesHeader';
+import Carousel from './Carousel';
+import 'swiper/swiper-bundle.css';
+import '../../style.css';
+import Hotel from './Hotel';
+import data from '../../constants/content';
```

Comment:
you need to use ```https://fe-student-api.herokuapp.com/api/hotels/popular``` api

Reply:
pls take a look at point 6,7
https://github.com/Intellect-Fox/fe-6/blob/main/module-react/lesson-6/homework.md